exo-setup/src/app-setup.ls

```diff
@@ -26,26 +26,26 @@ class AppSetup extends EventEmitter
       if service.location
         new ServiceSetup role: service.role, logger: @logger, config: root: path.join(process.cwd!, service.location)
           ..on 'output', (data) ~> @emit 'output', data
-    async.map-series setups, (-> &0.start &1)
-
-    docker-setups = for service in @services
-      new DockerSetup do
-        role: service.role
-        logger: @logger
-        config:
-          root: path.join(process.cwd!, service.location) if service.location
-          docker-image: service.docker-image
-
-
-      # Note: Windows does not provide atomic file operations,
-      #       causing file system permission errors when multiple threads read and write to the same cache directory.
-      #       We therefore run only one operation at a time on Windows.
-    operation = if process.platform is 'win32'
-      async.map-series
-    else
-      async.map
-    operation docker-setups, (-> &0.start &1), (err) ~>
-      @logger.log role: 'exo-setup', text: 'setup complete'
+    async.map-series setups, (-> &0.start &1), (err) ~>
```

Comment:
```ls
| err  =>  throw new Error err
```

But really this method does a lot of async things, so it should be asynchrounous. Meaning take in a `done` callback and call it when done, optionally providing an error message. This can be done in a separate PR.